include/copc-lib/io/reader.hpp

```diff
@@ -36,7 +37,7 @@ class Reader : public BaseIO
     std::vector<Node> GetAllChildren() { return GetAllChildren(VoxelKey::BaseKey()); }
 
   private:
-    std::istream &in_stream_;
+    std::istream *in_stream_{};
```

Comment:
remove the {}

Reply:
I think we need to keep it for the default constructor.

Reply:
Pointers don't need to be initialized